lib/modules/manager/flux/readme.md

```diff
@@ -26,6 +27,12 @@ Renovate can update `git` references from `GitRepository` resources.
 
 The `flux` manager only updates `GitRepository` fields that have a `tag` or `commit` key.
 
+### OCIRepository support
+
+Renovate can update `oci` references from `OCIRepository` resources.
+
+The `flux` manager only updates `OCIRepository` fields that have a `tag` or `digest` key.
```

Comment:
What happens when the `OCIRepository` fields have a `tag` **and** a `digest` key?

Reply:
According to the docs it seems digest will take precedent.

https://fluxcd.io/flux/components/source/ocirepositories/#digest-example